keyboards/spaceman/2_milk/config.h

```diff
@@ -17,8 +17,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0xB195
+#define VENDOR_ID       0x5350
+#define PRODUCT_ID      0x3225
```

Comment:
```suggestion
#define VENDOR_ID       0x5350 // "SP"
#define PRODUCT_ID      0x3225 // "2%"
```

Reply:
id prefer SB here

Reply:
... which would make this:

```suggestion
#define VENDOR_ID       0x5342 // "SB"
#define PRODUCT_ID      0x3225 // "2%"
```